src/main/java/kr/hhplus/be/server/ranking/controller/GoodsRankingController.java

```diff
@@ -0,0 +1,23 @@
+package kr.hhplus.be.server.ranking.controller;
+
+import kr.hhplus.be.server.ranking.domain.GoodsRankDto;
+import kr.hhplus.be.server.ranking.service.GoodsRankingService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/goods/ranking")
+@RequiredArgsConstructor
+public class GoodsRankingController {
+
+    private final GoodsRankingService goodsRankingService;
+
+    @PostMapping("/top10")
```

Comment:
```suggestion
    @GetMapping("/top10")
```
리소스 조회는 GET을 사용하는 것이 좋을것같습니다.